recipe/meta.yaml

```diff
@@ -31,8 +31,7 @@ requirements:
     - python
     - numpy
     - hdf5
-    - libgfortran  # [not win]
-    - libgfortran5  # [not win]
+    - {{ compiler('fortran') }}
```

Comment:
This needs to be in build. It will add the proper thing in host.

Reply:
Argh sorry I should have read more closely, will push momentarily!